src/rdkafka_queue.c

```diff
@@ -517,7 +517,35 @@ int rd_kafka_q_serve (rd_kafka_q_t *rkq, int timeout_ms,
 }
 
 
+int invalid_all_old_msgs_in_array(rd_kafka_op_t **rkos,
```

Comment:
Make static, use size_t for counts, use rdkafka naming convention:
`rd_kafka_purge_outdated_messages()`